sst.json

```diff
@@ -1,5 +1,5 @@
 {
   "name": "pickup",
-  "region": "us-east-2",
+  "region": "us-west-2",
```

Comment:
note: this only applies to local dev deploys run with `npm start` or `npx sst start`. The region is configured for staging and PR builds and prod in seed.run settings.

Reply:
Seed doesn't utilize this file? Bummer :(